readme.md

```diff
@@ -291,6 +292,25 @@ this disorder.
 
 [R: medicinet.com](https://www.medicinenet.com/munchausen_syndrome_by_proxy/article.htm)
 
+
+#### [Lima Syndrome](https://en.wikipedia.org/wiki/Stockholm_syndrome#Lima_syndrome)
+
+> A form of Stockholm Syndrome where abductors develop sympathy for their hostage
+
+Lima Syndrome was named after the abduction at the Japanese embassy in Lima, Peru, 
+in 1996 when memebers of a militant group took hostage people attending a party 
+at the official residence of the Japanese ambassador, but set most of them free after 
+a few hours because of sympathy towards the hostages. Some of the reasons it could happen 
+is when one or more kidnappers don't agree with others,  don't feel like hurting 
+the hostages or the hostage is prone to developing sympathy. This disorder has been
+used in media mutiple times including Beauty and the Beast, A Song of Ice and Fire, 
+The World is Not Enough and Grand Theft Auto V.
+
+[R: Lima Syndrome](http://tvtropes.org/pmwiki/pmwiki.php/Main/LimaSyndrome)
+
+[R: A Tale of Kidnapping-From Stockholm to Lima Syndrome](http://www.ipsnews.net/1996/07/peru-tale-of-a-kidnapping-from-stockholm-to-lima-syndrome/)
+
+
```

Comment:
You can put the scene in ASOIAF and Beauty and the Beast in which this syndrome was seen.

Reply:
Made all the changes on Lima Syndrome